modin/experimental/cloud/rpyc_proxy.py

```diff
@@ -618,7 +618,17 @@ def dtypes(self):
 
     DeliveringDataFrame = _deliveringWrapper(
         DataFrame,
-        ["groupby", "agg", "aggregate", "__getitem__", "astype", "drop", "merge"],
+        [
+            "groupby",
+            "agg",
+            "aggregate",
+            "__getitem__",
+            "astype",
+            "drop",
+            "merge",
+            "apply",
```

Comment:
that seems a little unsafe... but let's think about changing delivering implementation later